pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.sonian</groupId>
   <artifactId>elasticsearch-jetty</artifactId>
-  <version>1.7.3-BV</version>
+  <version>1.7.3-SNAPSHOT</version>
```

Comment:
Right now, we've been doing branch builds. I can also play with the release job that's already in jenkins and get that done, but figured we'd get through the CR's before I went ahead and did that.